Apps/WebClient/src/appsettings.json

```diff
@@ -32,21 +32,8 @@
             "Icon": "fas fa-address-card",
             "Hint": "bcsc",
             "disabled": "false"
-        },
-        {
-            "id": "IDIR",
-            "Name": "BC Government IDIR",
-            "Icon": "fas fa-user",
-            "Hint": "idir",
-            "disabled": "false"
-        },
-        {
-            "id": "KeyCloak",

```

Comment:
without these we wont be able to run functional tests on dev. Just make sure after deployment it works as expected.

Reply:
I did test it and seems to work. I'm not sure how to go about this

Reply:
we need to set those on the configuration for dev if they are going to be deleted here